OpenProblemLibrary/Michigan/Chap5Sec4/Q03.pg

```diff
@@ -42,7 +42,8 @@ loadMacros(
   "PGchoicemacros.pl",
   "PGgraphmacros.pl",
   "MathObjects.pl",
-  "PGcourse.pl"
+  "PGcourse.pl",
+  "niceTables.pl"
```

Comment:
`PGcourse.pl` should always be the last macro in the list.

Reply:
Didn't know that, thank you!